lib/libvfio-user.c

```diff
@@ -1832,6 +1840,10 @@ vfu_setup_region(vfu_ctx_t *vfu_ctx, int region_idx, size_t size,
 
     assert(vfu_ctx != NULL);
 
+    /*
```

Comment:
make this an spec issue instead?

Reply:
sure

Reply:
https://github.com/nutanix/libvfio-user/issues/699